sphinx/source/docs/user_guide/compat.rst

```diff
@@ -75,15 +65,21 @@ requires only one line of code in HoloViews:
  :align: center
 
 Adding overlaid plots, slider widgets, selector widgets, selection
-tools, and tabs is similarly straightforward. HoloViews objects can
-also be rendered using a Matplotlib-based backend, which allows SVG or
-PDF output not currently available for native Bokeh plots. See the
-Holoviews `Bokeh_Backend`_ tutorial for more details.
+tools, and tabs is similarly straightforward.
 
+Further synergy
+---------------
```

Comment:
I'd like propose an organization change here. By itself, Datashader and Bokeh do not have much of an integration story. Datashader just generates images and has no direct Bokeh integation, and Bokeh does not know about Datashader at all. The story is Holoviews +Datashader. 

So I would propose: Rename "Further synergy" to "Datashader" and move all the datashader paragraphs above here (above the content below)

thoughs @jbednar ?

Reply:
When this was written, I think Datashader still had direct Bokeh integration, but that was not very useful, and nowadays I don't know of anyone using Datashader directly with Bokeh. So, sure!